From d8ab989f1dd165289074530bd99c61d1d12556ae Mon Sep 17 00:00:00 2001
From: rth <rth@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Thu, 24 Jul 2014 18:30:44 +0000
Subject: [PATCH] AArch64: emit .note.GNU-stack

        2014-07-24  Kyle McMartin  <kyle@redhat.com>
        * config/aarch64/aarch64-linux.h (TARGET_ASM_FILE_END): Define.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@213029 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/config/aarch64/aarch64-linux.h |    2 ++
 2 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/gcc/config/aarch64/aarch64-linux.h b/gcc/config/aarch64/aarch64-linux.h
index a8f0771..57893fc 100644
--- a/gcc/config/aarch64/aarch64-linux.h
+++ b/gcc/config/aarch64/aarch64-linux.h
@@ -44,4 +44,6 @@
     }						\
   while (0)
 
+#define TARGET_ASM_FILE_END file_end_indicate_exec_stack
+
 #endif  /* GCC_AARCH64_LINUX_H */
-- 
1.7.1

